src/main/resources/view/DarkTheme.css

```diff
@@ -153,6 +153,13 @@
     -fx-text-fill: white;
 }
 
+.select-view {
+    -fx-background-color: transparent;
+    -fx-font-family: "Segoe UI Light";
```

Comment:
Maybe a [System Font Stack](https://modernfontstacks.com/) might be good here?

If I'm not wrong Segoe UI is only a windows font haha

Reply:
Actually, seems like the original code used Segoe UI as the font. Not sure if it works as intended though on MacOS!